docs/getting-started/gantt-chart-introduction/README.md

```diff
@@ -30,15 +30,15 @@ To find out more about the functionalities for Gantt charts, please visit our [u
 
 With Gantt charts in OpenProject you can create and manage a project plan and share the information with your team. You can schedule your tasks and visualize the required steps to complete your project. As a project manager you are directly informed about delays in your project and can act accordingly.
 
-The dynamic Gantt chart in OpenProject displays the phases and milestones in your project as well as relationships between them. Each element in the timeline has both a start as well as a finish date so you are always informed about the current status.
+The dynamic Gantt chart in OpenProject displays the phases, tasks and milestones in your project as well as relationships between them. The elements for phases and tasks each have a start and end date, so you are always informed about the current status. The milestone only appears as a fixed point in time.
 
 ## Activate the Gantt chart view
 
-To open the Gantt chart view in OpenProject, the [work packages module needs to be activated](../projects/#advanced-project-settings) in the project settings.
+To open the Gantt chart view in OpenProject, the [work packages module needs to be activated](../../user-guide/activity) in the project settings.
 
 Within your project menu, navigate to the work packages module. Select the **Gantt chart view** in the work package list with the button on the top right.
 
-![Activate-Gantt-chart](Activate-Gantt-chart.png)
+![Bildschirmfoto 2022-02-23 um 17.45.10](/Users/jankeppler/Desktop/Bildschirmfoto 2022-02-23 um 17.45.10.png)
```

Comment:
This image will not be shown, since it is on your computer only. Please include the file into the folder, add it to git and rename it, so there are no spaces in it. Please take a look [here](https://www.openproject.org/docs/development/contribution-documentation/documentation-style-guide/#save-images) for more info.
Please use a proper image description.